components/SpeakersSection.js

```diff
@@ -46,23 +47,41 @@ export default function TicketsSection () {
         }
         .speakers-section {
           text-align: center;
+          width: 1060px;
```

Comment:
nit: This line makes div overflow on screen width 800~1000px. I think it is fine to remove it.